lib/mix/tasks/users/import.ex

```diff
@@ -48,13 +48,13 @@ defmodule Mix.Tasks.Cadet.Users.Import do
       csv_stream = path |> File.stream!() |> CSV.decode(strip_fields: true)
 
       for {:ok, [name, username, group_name]} <- csv_stream do
-        with {:ok, group = %Group{}} <- Course.get_or_create_group(group_name),
+        with {:ok, _group = %Group{}} <- Courses.get_or_create_group(group_name),
```

Comment:
I assume this file is incomplete, right? (It's missing the course ID)

Reply:
Yes, this part will be updated when we deal with the onboarding process.